src/replay.rs

```diff
@@ -16,12 +16,8 @@ use super::util;
 pub fn replay(args: &Args) {
     assert!(args.cmd_replay);
 
-    let cargo_toml_path = Path::new(&args.flag_cargo);
-
-    if !cargo_toml_path.exists() || !cargo_toml_path.is_file() {
```

Comment:
I guess it's not super important, but it seems worthwhile to keep this printout in place...